processor/parse/flavored/callout.js

```diff
@@ -1,5 +1,30 @@
+// @note: We'd like to allow any emoji to match, but included in the emoji
+// character set is [#*0-9].
+// https://www.unicode.org/Public/13.0.0/ucd/emoji/emoji-data.txt
 // eslint-disable-next-line unicorn/no-unsafe-regex
-const rgx = /^> ?((?:\u2139|\u2049|\u2757|\u203C|\u00a9|\u00ae|[\u2000-\u3300]|\ud83c[\ud000-\udfff]|\ud83d[\ud000-\udfff]|\ud83e[\ud000-\udfff])+)(?: {0,}(.+))?\n((?:>(?: .*)?\n)*)/;
+const rgx = /^> ?((?![#*0-9])\p{Emoji})(?: {0,}(.+))?\n((?:>(?: .*)?\n)*)/u;
+
+const themes = {
+  '\uD83D\uDCD8': 'info',
+  '\u26A0\uFE0F': 'warn',
+  '\uD83D\uDEA7': 'warn',
+  '\uD83D\uDC4D': 'okay',
+  '\u2705': 'okay',
+  '\u2757': 'error',
+  '\u2757\uFE0F': 'error',
+  '\uD83D\uDED1': 'error',
+  '\u2049\uFE0F': 'error',
+  '\u203C\uFE0F': 'error',
+  '\u2139\uFE0F': 'info',
+  '\u26A0': 'warn',
+};
+
+const icons = Object.entries(themes).reduce((acc, [icon, theme]) => {
```

Comment:
Why transform the array here, as opposed to having the themes be a `theme=>[icon]` mapping from the start?

Reply:
The `themes` mapping is used elsewhere during parsing. Figured I needed both.

https://github.com/readmeio/markdown/pull/219/files/62c779fb59b7352cca4436d9580c0873b7c30c2a#diff-7cc68b0d2e3b47c833b432fdc9858c4a1641b17ca441f1a8e7b01542232b5700R39